js/main.js

```diff
@@ -1,6 +1,173 @@
+      <div class="card-content">\n
+        <h2 class="title-post-2">${proj.title}</h2>\n
+        <p class="tema-2">${proj.description}</p>\n
+        <ul class="btn-group">\n
+        ${technologies}
+        </ul>\n
+      </div>\n
+      <button class="link-btn-2 fill-h" onclick="seeProject('${project}')">See Project</button>\n
+    </div>\n`;
+    } else {
+      for (let i = 0; i < projTech.length; i += 1) {
+        technologies += `\t\t\t\t\t\t<button class="p-lang">${projTech[i]}</button>\n`;
+      }
+      topCard += `
+      \t\t\t\t<div class="fit-container g-1">\n
+      \t\t\t\t\t<img class="fit-container" src="${proj.imgb}" alt="picture">\n
+      \t\t\t\t</div>\n
+      \t\t\t\t<div class="fit-container g-1">\n
+      \t\t\t\t\t<h2 class="title-post">${proj.title}</h2>\n
+      \t\t\t\t\t<p class="tema-2">${proj.description}</p>\n
+      \t\t\t\t\t<div class="btn-group g-1 spaced">\n
+      ${technologies}
+      \t\t\t\t\t</div>\n
+      \t\t\t\t\t<button class="link-btn fit" onclick="seeProject('${project}')">See Project</button>\n
+      \t\t\t\t</div>\n\n`;
+    }
+  }
+  document.getElementById('top-card').innerHTML = topCard;
+  document.getElementById('card-group').innerHTML = cardGroup;
+}
+
 function displayMenu() {
   document.querySelector('#menu-button').classList.toggle('hidden');
   document.querySelector('#wide-menu').classList.toggle('hidden');
+  document.querySelector('body').classList.toggle('noscroll');
+}
+
+// eslint-disable-next-line
+function seeProject(id) {
+  const project = projects[id];
+  let technologies = '';
+  for (let i = 0; i < project.technologies.length; i += 1) {
+    technologies += `\t\t\t\t\t\t<button class="p-lang g-1">${project.technologies[i]}</button>`;
+  }
+  document.querySelector('#card-modal').classList.toggle('hidden');
+  document.querySelector('#modal-title').textContent = project.title;
+  document.getElementById('modal-tecs').innerHTML = technologies;
+  document.querySelector('#modal-img').textContent = project.imgb;
+  document.querySelector('#modal-desc').textContent = project.description;
+  document.getElementById('open-links').innerHTML = `\t\t\t\t\t\t\t<button class="link-btn flex" onclick="openURL('${project.linkLive}')">See Live <img src="./icons/Icon-Export.png" alt="Live"></button>\n
+  \t\t\t\t\t\t\t<button class="link-btn flex" onclick="openURL('${project.linkSource}')">See Source <img src="./icons/Icon-GitHub3.png" alt="GitHub"></button>\n`;
+}
+
+// eslint-disable-next-line
+function openURL(url) {
+  window.open(url, '_blank').focus();
+}
+
+function closeModal() {
+  document.querySelector('#card-modal').classList.toggle('hidden');
 }
+
+const email = document.getElementById('email');
+
+document.querySelector('#email').addEventListener('keyup', (e) => {
+  const regx = /[A-Z]/g;
+  if (regx.test(email.value)) {
+    e.preventDefault();
+    window.alert(`${email.value} es texto inválido`); // eslint-disable-line
```

Comment:
- [ ] Nice work done on your email validation, kindly check the validation works when you first type, but when I dismiss the alert box and click submit again it goes successfully.
- [ ] As a requirement the error message should show up close to the "submit" button not as an alert.